test/configCases/entry/depend-on-simple/webpack.config.js

```diff
@@ -1,3 +1,7 @@
+/** @typedef {import("../../../../").Compiler} Compiler */
```

Comment:
Maybe add alias to typescript for `~`? e.g.
```
"paths": {
			"~/*": ["./lib/*"]
		}
```

Reply:
`/** @typedef {import("webpack").Compiler} Compiler */`

would be cool. We can do that in a separate PR, I spend too long on the PR already, need a break of typings...